docs/core/setup/index.md

```diff
@@ -0,0 +1,453 @@
+
+---
+
+::: zone-end
+
+::: zone pivot="os-windows"
+
+## Install with Visual Studio
+
+Visual Studio installs the latest .NET Core SDK tools, and one or more runtimes so you can target your app to run on the .NET Core runtime version of your choice.
+
+Selecting any of the managed workloads for **Web**, **Desktop**, or **Azure development**, will install the .NET Core development tools and runtime. Specifically choosing the **.NET Core** workload also installs .NET Core.
+
+[![Windows Visual Studio 2019 with .NET Core workload](media/index/windows-install-visual-studio-2019.png)](media/index/windows-install-visual-studio-2019.png#lightbox)
+
+After you've installed Visual Studio, create your first app by following the [C# Hello World tutorial](../tutorials/with-visual-studio.md) or the [Visual Basic Hello World tutorial](../tutorials/vb-with-visual-studio.md).
+
+::: zone-end
+
+::: zone pivot="os-macos"
+
+## Install with Visual Studio for Mac
+
+Visual Studio for Mac installs the .NET Core SDK when the **.NET Core workload** is selected. To get started with .NET Core development on macOS, see [Install Visual Studio 2019 for Mac](https://docs.microsoft.com/visualstudio/mac/installation?view=vsmac-2019).
+
+[![macOS Visual Studio 2019 for Mac with .NET Core workload feature](media/index/mac-install-selection.png)](media/index/mac-install-selection.png#lightbox)
+
+After you've installed Visual Studio for Mac, create your first app by following the [Get started on macOS](../tutorials/using-on-mac-vs.md) tutorial.
+
+::: zone-end
+
+## Install with Visual Studio Code
+
+Visual Studio Code is a powerful and lightweight source code editor that runs on your desktop. VS Code is available for Windows, macOS, and Linux.
+
+While VS Code doesn't come with .NET Core support, adding .NET Core support is simple.
+
+01. [Download and install Visual Studio Code](https://code.visualstudio.com/Download).
+01. [Download and install the .NET Core SDK](https://dotnet.microsoft.com/download/dotnet-core/3.0).
+01. [Install the C# extension from the VS Code marketplace](https://marketplace.visualstudio.com/items?itemName=ms-vscode.csharp).
+
+After you've installed .NET Core support for VS Code, create your first app by following the [Using .NET Core in Visual Studio Code](https://code.visualstudio.com/docs/languages/dotnet) tutorial.
+
+::: zone pivot="os-windows"
+
+## Install with PowerShell automation
+
+The [dotnet-install scripts](../tools/dotnet-install-script.md) are used for a non-admin install of the CLI toolchain and the shared runtime. You can download the script from [dotnet-install script reference page](../tools/dotnet-install-script.md).
+
+The script defaults to installing the latest "LTS" version, which is currently .NET Core 2.1. To install the current release of .NET Core (3.0), run the script with the following switch:
+
+```powershell
+dotnet-install.ps1 -Channel LTS
```

Comment:
This snippet doesn't seem to match the description

>To install the current release of .NET Core (3.0)

I would expect the Channel to be set to `3.0`